ec/src/models/short_weierstrass/mod.rs

```diff
@@ -105,6 +105,14 @@ pub trait SWCurveConfig: super::CurveConfig {
         res
     }
 
+    /// Default implementation for multi scalar multiplication
+    fn model_msm_bigint(
+        bases: &[Affine<Self>],
+        bigints: &[<Self::ScalarField as PrimeField>::BigInt],
+    ) -> Projective<Self> {
+        VariableBaseMSM::unhooked_msm_bigint(bases, bigints)
```

Comment:
```suggestion
        VariableBaseMSM::msm_bigint(bases, bigints)
```
That should be enough no?

Reply:
It seems like there is no way reference the default implementation of a function in a trait explicitly. Once we override it, only the overridden method is available. That's the underlying problem.

It might look nicer though to just rename `unhooked_msm_bigint` into `default_msm_bigint`?

Reply:
I renamed `unhooked_msm_bigint` into `default_msm_bigint`.